CRM/Contact/BAO/Query.php

```diff
@@ -6889,7 +6889,7 @@ protected function getMetadataForField($fieldName) {
     if (!empty($field) && empty($field['name'])) {
       // standardising field formatting here - over time we can phase out variants.
       // all paths using this currently unit tested
-      $field['name'] = CRM_Utils_Array::value('field_name', $field, CRM_Utils_Array::value('idCol', $field, $fieldName));
+      $field['name'] = $field['field_name'] ?? $field['idCol'] ?? $fieldName;
```

Comment:
I don't know if prevents issue but I would normally use parentheses on this - ie after the first ??

Reply:
@eileenmcnaughton no need for parenthesis because all of these operators are the same (`??`) so they all have the same precedence.

Reply:
hmm - not even for reability?

Reply:
I'm not sure what you mean. The code is basically saying:
```
$that = $1 or $2 or $3;
```
It's either 1 or 2 or 3, in that order. Grouping any two of them with parenthesis wouldn't make sense because there are no groupings here, it's just linear.

Reply:
hmmm OK